assignments/arrays.js

```diff
@@ -63,32 +74,72 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+//console.log(`Car 33 is a *2011* *Jeep* *Wrangler*`);
+
+function findCar(id) {
+  let a = id - 1;
+  let b = inventory[a];
+  return (
+    "The car with ID tag " +
+    id +
+    " has a year of " +
+    b["car_year"] +
+    " and is a " +
+    b["car_make"] +
+    " " +
+    b["car_model"] +
+    "."
+  );
+}
+console.log(findCar(33));
```

Comment:
Excellent use of a function to provide declarative code with purpose of the challenge!